readme.txt

```diff
@@ -98,6 +98,46 @@ Please note that our support for the checkout block is still experimental and th
 
 == Changelog ==
 
+= 5.6.0 - 2023-03-15 =
+* Add - Add a component that provides a tour experience.
+* Add - Add a notice for Norwegian merchants using WCPay to update WooCommerce to at least 7.5
+* Add - Added WC_Payments_Customer_Service_API in WCPayCore
```

Comment:
In `changelog.txt`, it's `WCPay\Core` but here, it's `WCPayCore`. Worth creating issue to fix the changelog workflow @jbordonado ?

Reply:
You can add a comment about this in https://github.com/Automattic/woocommerce-payments/issues/5647

Reply:
Commented https://github.com/Automattic/woocommerce-payments/issues/5647#issuecomment-1469710845.